audio/midi_drivers/FluidSynthMidiDriver.cpp

```diff
@@ -64,13 +68,22 @@ int FluidSynthMidiDriver::open() {
 	if (!soundfont.empty())
 		soundfonts.push_back(soundfont);
 
+	_settings = new_fluid_settings();
+
 	if (soundfonts.empty()) {
-		perr << "FluidSynth requires a 'fluidsynth_soundfont' setting" << std::endl;
-		return -2;
+		char *default_soundfont;
+		if (getStr("synth.default-soundfont", &default_soundfont) == FLUID_OK &&
+		    default_soundfont && default_soundfont[0] != 0) {
+			soundfonts.push_back(default_soundfont);
+			fluid_free(default_soundfont);
+			perr << "FluidSynth without a 'fluidsynth_soundfont' setting, using default soundfont" << std::endl;
```

Comment:
If I am understanding this correctly, please change to something like

`There was no 'fluidsynth_soundfont' setting in Exult.cfg; falling back to default Fluidsynth soundfont`